server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -975,7 +981,23 @@ private void loadFile(File file, OperateFilePlan plan) throws QueryProcessExcept
         createSchemaAutomatically(chunkGroupMetadataList, schemaMap, plan.getSgLevel());
       }
 
-      StorageEngine.getInstance().loadNewTsFile(tsFileResource);
+      List<TsFileResource> splitResources = new ArrayList();
+      try {
+        tsFileResource.getTimePartitionWithCheck();
```

Comment:
How about we add a method that  judgment  whether it contains multiple partitions？

Reply:
That is another method, I will add another method to judge whether it contains multiple partitions.